Gemfile

```diff
@@ -8,7 +8,9 @@ ENV['BOLT_DISABLE_ANALYTICS'] = 'true'
 gemspec
 
 # Bolt server gems are managed here not in the gemspec
+gem "bigdecimal", '~> 1.3.5'
```

Comment:
Alpine provides packages for these, ruby-bigdecimal and ruby-json. Not sure whether it's better to install packages are add gem dependencies for things that are commonly in core ruby.

Reply:
Switching to using `ruby-bigdecimal` and `ruby-json`, and using non-dev versions of ruby and openssl gets my image down to 91.5MB. So I think it's worth going that route.